packages/gatsby-theme-docs/src/components/top-menu.js

```diff
@@ -237,6 +252,60 @@ const TopMenu = (props) => {
 
 TopMenu.propTypes = {
   centered: PropTypes.bool,
+  closeTopMenu: PropTypes.func,
 };
 
+const MenuTop = styled.div`
+  display: inline-flex;
+`;
+
+const MenuBottom = styled.div`
+  background-color: ${designSystem.colors.light.surfaceSecondaryTopMenu};
+  display: inline-flex;
+  border-bottom-left-radius: 10px;
+  border-bottom-right-radius: 10px;
```

Comment:
Same as above, there are several places in this file where we could use the design system